cEP-0005.md

```diff
@@ -85,6 +85,16 @@ files = **.(c|h)
 ignore_aspects = redundancy  # Exclusive
 ```
 
+aspect name could be writen by its fullname (`Root.Redundancy.Clone`) or
+partial name (`Redundancy.Clone` or even `Clone`) and are case insensitive.
+Note that writing by partial name could result in ambiguity. Thus the desired
+aspect must be written by its fullname.
+
+In case of multiple aspect has a same taste name, ambiguity will be resolved
+by prefixing the taste name with its aspect name. For example, `max_length`
+is a taste under `Shortlog` and `LineLength`. Thus defining the taste must
+be written by `Shortlog.max_length = 50`.
```

Comment:
* *multiple aspects have ...*
* *is a taste of ...*
* *Thus the `Shortlog` taste would be defined like ...*